### PR TITLE
Packaging Updates & Bug Fixes

### PR DESCRIPTION
**Pull request by KPrasch**
- Updates version freeze to `bytsetring-splitter == 1.0.0a4`
- Corrects typo in project description
- Use `setuptools.find_packages` for umbral package discovery
- Update `Pipfile.lock` `requirements.txt` and `dev-requirements.txt`
- Use file-based requirements in `setup.py` (Generated via `pipenv`)

### CODE REVIEW

**Comment by cygnusv** [comment]
Can you change here the dependency of `bytestringSplitter` to `bytestring-splitter`?

File: setup.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def run(self):
 INSTALL_REQUIRES = ['setuptools',
                     'cryptography>=2.3',
                     'pynacl',
-                    'byteStringSplitter',
+                    'bytestring-splitter',
```
Comment (cygnusv) [inline setup.py]: It's necessary to update the Pipfile and Pipfile.lock

Comment (KPrasch) [reply setup.py]: Right - Working on this.

Comment (KPrasch) [reply setup.py]: :white_check_mark:

**Pull request merged**